File: 백준/Bronze/3003. 킹， 퀸， 룩， 비숍， 나이트， 폰/킹， 퀸， 룩， 비숍， 나이트， 폰.java
import java.io.BufferedReader;
import java.io.BufferedWriter;
import java.io.IOException;
import java.io.InputStreamReader;
import java.io.OutputStreamWriter;
import java.util.StringTokenizer;

public class Main {

	public static void main(String[] args) throws IOException {
		
		BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
		BufferedWriter bw = new BufferedWriter(new OutputStreamWriter(System.out));
		
		StringTokenizer st = new StringTokenizer(br.readLine());
		
		int king = 1;
		int queen = 1;
		int look = 2;
		int bishop = 2;
		int knight = 2;
		int pawns = 8;
		
		king = king - Integer.parseInt(st.nextToken()); 
		queen = queen - Integer.parseInt(st.nextToken()); 
		look = look - Integer.parseInt(st.nextToken()); 
		bishop = bishop - Integer.parseInt(st.nextToken()); 
		knight = knight - Integer.parseInt(st.nextToken()); 
		pawns = pawns - Integer.parseInt(st.nextToken());
		
		bw.write(king + " " + queen + " " + look + " " + bishop + " "+ knight + " " + pawns);
		
		bw.flush();
		bw.close();
		br.close();
	}
		
}

File: 백준/Bronze/3003. 킹， 퀸， 룩， 비숍， 나이트， 폰/킹， 퀸， 룩， 비숍， 나이트， 폰.cc
#include <iostream>
using namespace std;

int main()
{
	ios_base::sync_with_stdio(false);
	cout.tie(0);

	int king, queen, look, bishop, knight, pawns;

	cin >> king >> queen >> look >> bishop >> knight >> pawns;

	cout << 1 - king << " " << 1 - queen << " " << 2 - look << " "
		<< 2 - bishop << " " << 2 - knight << " " << 8 - pawns;

	return 0;
}